bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -501,6 +501,7 @@ public function doGetGridColumnConfig(Request $request, Config $config, $isDelet
                             'key' => $key,
                             'type' => 'system',
                             'label' => $key,
+                            'locked' => $sc['locked'],
```

Comment:
please set lock property for saved calculatedcolumns(operators etc.) here:
https://github.com/pimcore/pimcore/blob/035465f49002e1278d134f0a6ab51b7414a6c4bc/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php#L797

and also set for container types(bricks, classificationstore etc.) somewhere here:
https://github.com/pimcore/pimcore/blob/035465f49002e1278d134f0a6ab51b7414a6c4bc/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php#L510

Reply:
I have applied this now also to the other fields like classificationstore, objectbricks and any operator.